js/data/data.js

```diff
@@ -0,0 +1,124 @@
+var Firebase = require('firebase');
+var FirebaseUtils = require('../utils/firebase');
+
+// TODO put the uri in a constant / env var
+var FirebaseRef = new Firebase('https://quack.firebaseio.com');
+
+module.exports = {
+  create: function(resourceType, options) {
```

Comment:
Is 'options' the right name for this variable? Seems like "data" or "payload" makes more sense.